Shop.DAL/Contexts/ShopContext.cs

```diff
@@ -24,6 +25,14 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
 
       protected override void OnModelCreating(ModelBuilder modelBuilder)
       {
+         base.OnModelCreating(modelBuilder);
+
+         modelBuilder.ApplyConfiguration(new CategoryConfiguration());
+         modelBuilder.ApplyConfiguration(new GameConfiguration());
+         modelBuilder.ApplyConfiguration(new OrderConfiguration());
+         modelBuilder.ApplyConfiguration(new OrderItemConfiguration());
+         modelBuilder.ApplyConfiguration(new RoleConfiguration());
+         modelBuilder.ApplyConfiguration(new UserConfiguration());
```

Comment:
you can use  modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());